core/src/main/scala/cats/SemigroupK.scala

```diff
@@ -31,9 +31,9 @@ import simulacrum.{op, typeclass}
   /**
    * Compose two SemigroupK intsances.
    */
-  def composedWith[G[_]: SemigroupK]: SemigroupK[λ[α => F[G[α]]]] =
+  def compose[G[_]: SemigroupK]: SemigroupK[λ[α => F[G[α]]]] =
```

Comment:
We have [Alternative](https://github.com/non/cats/blob/master/core/src/main/scala/cats/Alternative.scala) which extends both `Applicative` and `MonoidK`. Not that calling it `composedWith` is a particularly elegant solution, but do we run the risk of overloading ambiguities/annoyances with both `compose`s being on the same instance?